packages/charts/examples/components/demo/chart-types/bar/chart-docs-bar.component.html

```diff
@@ -102,7 +102,7 @@ <h2>Time Interval</h2>
     intervals, you can set the scale for the x-axis to an instance of  <a href="../classes/TimeIntervalScale.html" target="_blank"><code>TimeIntervalScale</code></a> and pass
     the desired interval of type moment <a href="https://momentjs.com/docs/#/durations/" target="_blank"><code>Duration</code> </a> to the <a href="../classes/TimeIntervalScale.html" target="_blank"><code>TimeIntervalScale</code></a>'s constructor.
 </p>
-<nui-example-wrapper filenamePrefix="bar-chart-time-interval"
+<nui-example-wrapper filenamePrefix="bar-chart-time-interval.example"
```

Comment:
example wrapper was pulling in another set of files that start with `bar-chart-time-interval`, so I added this to fix that.